code/modules/reagents/reagents/drugs.dm

```diff
@@ -164,6 +164,8 @@
 	..()
 	M.add_chemical_effect(CE_PULSE, 1)
 	M.add_chemical_effect(CE_PAINKILLER, 5 * effect_multiplier)
+	if(M.stats.getPerk(PERK_CHAINGUN_SMOKER))
```

Comment:
We should probably add small brute and burn effect here as well.

Reply:
Are you sure of this?


Are you sure of this?
Would the effect apply to everyone in general or only to those who have perk associated with smoking?

Reply:
Only to people with perks of course.

Tric is cheap anyway. Making a perk that make nicotine acts like tric will not impact the balance.